systems-state-manager/nisystemsstate.yml

```diff
@@ -0,0 +1,999 @@
+      description:
+        type: string
+        description: Description of the State
+        example: crio9064 Setup State
+      distribution:
+        $ref: '#/definitions/StateDistribution'
+      architecture:
+        $ref: '#/definitions/StateArchitecture'
+      feeds:
+        type: array
+        items:
+          $ref: '#/definitions/Feed'
+      packages:
+        type: array
+        items:
+          $ref: '#/definitions/Package'
+    required:
+      - name
+      - distribution
+      - architecture
+
+  StatePatchRequest:
+    description: System state object defining which attributes have to be replaced.
+    title: State Patch Request
+    x-ni-sparse: true
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the State.
+        example: crio9064State
+      description:
+        type: string
+        description: Description of the State.
+        example: crio9064 Setup State
+      distribution:
+        $ref: '#/definitions/StateDistribution'
+      architecture:
+        $ref: '#/definitions/StateArchitecture'
+      feeds:
+        type: array
+        items:
+          $ref: '#/definitions/Feed'
+      packages:
+        type: array
+        items:
+          $ref: '#/definitions/Package'
+      changeDescription:
+        type: string
+        description: Description of the change operation.
+        example: I've changed the state
+
+  StateResponse:
+    description: System state object.
+    title: State response
+    type: object
+    properties:
+      id:
+        type: string
+        description: The ID of the state.
+        example: 5bc4463af0f06f4585003ae6
+      name:
+        type: string
+        description: The name of the state.
+        example: crio9064State
+      fileName:
+        type: string
+        description: The file name of the state.
+        example: crio9064State
+      description:
+        type: string
+        description: The description of the state.
+        example: crio9064 Setup State
+      distribution:
+        $ref: '#/definitions/StateDistribution'
+      architecture:
+        $ref: '#/definitions/StateArchitecture'
+      createdTimestamp:
+        description: ISO-8601 formatted timestamp specifying the state creation date.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      lastUpdatedTimestamp:
+        description: ISO-8601 formatted timestamp specifying the last date that the state has been updated.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      feeds:
+        type: array
+        items:
+          $ref: '#/definitions/Feed'
+      packages:
+        type: array
+        items:
+          $ref: '#/definitions/Package'
+      containsExtraOperations:
+        type: boolean
+        description: Whether the state containts extra operations beside feeds and packages or not.
```

Comment:
"Whether the state contains extra operations in addition to feeds and packages."?